### PR TITLE
Smc3 container image vulnerabilities scans general overview

### PR DESCRIPTION
**Pull request by detaybey**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

File: pkg/dashboard/dashboard.go [hunk 1]
```diff
@@ -183,6 +185,15 @@ func GetRouter(c config.Configuration, port int, basePath string) *mux.Router {
 		imageScanDetailsHandler(w, r, c, basePath, &scanResult)
 	})
 
+	router.HandleFunc("/overview", func(w http.ResponseWriter, r *http.Request) {
```
Comment (v1r7u) [inline pkg/dashboard/dashboard.go]: please, change path to `/images-overview`

File: pkg/dashboard/assets/js/overview.js [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+            sublist += '<li>';
+            sublist += ' <div style="float:right;">' + rowText + '</div>';
+            if(rowText === 'No Issues') {
+                sublist += '<i class="fas fa-check noissues-icon"></i>';
+            }else if (rowText === 'No Data') {
+                sublist += '<i class="fas fa-times nodata-icon"></i>';
+            }else {
+                sublist += '<i class="fas fa-exclamation-triangle warning-icon"></i>';
+            }
+            sublist += shortenImageName(resultGroup.images[j].image);
```
Comment (v1r7u) [inline pkg/dashboard/assets/js/overview.js]: Could you also add a link to a single-image page?

**Review by v1r7u — CHANGES_REQUESTED** [review]

**Comment by detaybey** [comment]
-[ok] repeating images for namespaces
-[ok] change url of overview with image-overview
-[ok] create links for image/nginx (try using (?) icons for links)
-[ok] add a search box
-[ok] sort groups by severity as well

File: pkg/dashboard/assets/js/overview.js (hunk 2 above)
Comment (detaybey) [reply pkg/dashboard/assets/js/overview.js]: done.

**Pull request merged**